File: Python/Rosliny/Trawa.py
from Roslina import Roslina


class Trawa(Roslina):
    def __init__(self, x, y, moj):
        self.sila = 0
        self.inicjatywa = 0
        self.symbol = 't'
        self.nazwa = "trawa"
        self.wiek = 0
        self.x = x
        self.y = y
        self.moj = moj
        self.czyZyje = True

    def generator(self, x, y):
        return Trawa(x, y, self.moj)

File: Java/src/pl/eco/roslina/Trawa.java
package pl.eco.roslina;

import pl.eco.Roslina;
import pl.eco.Swiat;

public class Trawa extends Roslina {
    public Trawa(int x, int y, Swiat moj) {
        sila = 0;
        inicjatywa = 0;
        nazwa = "trawa";
        symbol = 't';
        wiek = 0;
        this.x = x;
        this.y = y;
        this.moj = moj;
    }
    public Trawa(int x, int y,Swiat moj,int wiek){
        sila = 0;
        inicjatywa = 0;
        nazwa = "trawa";
        symbol = 't';

        this.wiek = wiek;
        this.x = x;
        this.y = y;
        this.moj = moj;
    }
}
